src/native/fs/fs_napi.cpp

```diff
@@ -86,11 +86,13 @@ struct FSWorker : public Napi::AsyncWorker
         , _uid(ThreadScope::orig_uid)
         , _gid(ThreadScope::orig_gid)
         , _errno(0)
+        , _warn_threshold_ms(100)
     {
         Napi::Object config = info[0].As<Napi::Object>();
         if (config.Has("uid")) _uid = config.Get("uid").ToNumber();
         if (config.Has("gid")) _gid = config.Get("gid").ToNumber();
         if (config.Has("backend")) _backend = config.Get("backend").ToString();
+        if (config.Has("warn_threshold_ms")) _warn_threshold_ms = config.Get("warn_threshold_ms").ToNumber();
```

Comment:
I wonder - if we send {warn_threshold_ms: undefined} does this condition evaluate to true or false?
We want to get false in that case, but not sure that Has() does that..

Reply:
I tested it and when we pass {warn_threshold_ms: undefined}  we will keep `_warn_threshold_ms(0)`
I guess that is what we are expecting.